CONTRIBUTING.md

```diff
@@ -29,31 +29,10 @@ the mailing list.
 
 If you start working on an issue, please assign yourself to it and mark the issue as "in progress".
 
-## Linking branches to JIRA
-
-If you include the JOINDIN-nnn issue number in the commit message, then JIRA will pick it up and display your branch on the issue.
-
-## JIRA Smart Commits
-
-If you want have your commit messages perform actions in JIRA. JIRA
-has [documentation](https://confluence.atlassian.com/display/FISHEYE/Using+Smart+Commits)
-on their extensive list of commands, but here is an example of what
-you can use in your commit message:
-
-```
-JOINDIN-<number> #close <message>
-```
-
-An example would be:
-
-```
-JOINDIN-445 #close Fixed.
-```
-
 ## PHP Version
 
 The current PHP version for joind.in is PHP 5.6.
 
 ## Code Style
 
-All PHP code follows [psr-2](http://www.php-fig.org/psr/psr-2/). One way to help ensure that your submitted code is PSR-2 is to run [PHP Code Sniffer]() against PSR-2 and the [.editorconfig](http://editorconfig.org/) plugin for your editor before submitting.
+All PHP code follows [psr-2](http://www.php-fig.org/psr/psr-2/). One way to help ensure that your submitted code is PSR-2 is to run [PHP Code Sniffer](https://github.com/squizlabs/PHP_CodeSniffer) against PSR-2 and the [.editorconfig](http://editorconfig.org/) plugin for your editor before submitting.
```

Comment:
Shouldn't we give the exact commandline for easier reference here? It should be something like `phpcs --standard psr2 .` so people know exactly what to do? Alternatively add a composer-script that does exactly that so people can "simply" run `composer checkstyle` and everything else is taken care of. Perhaps including downloading the phpcs-phar-file...